OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -1,61 +1,18 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
```suggestion
diff --git a/OpenRA.Game/OpenRA.Game.csproj b/OpenRA.Game/OpenRA.Game.csproj
index 38ad054123..7393d7388f 100644
--- a/OpenRA.Game/OpenRA.Game.csproj
+++ b/OpenRA.Game/OpenRA.Game.csproj
@@ -3,7 +3,7 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net461</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Optimize>true</Optimize>
+    <Optimize>false</Optimize>
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <LangVersion>5</LangVersion>
     <DebugSymbols>true</DebugSymbols>
@@ -30,6 +30,9 @@
     <StartAction>Project</StartAction>
     <StartArguments>Game.Mod=ts</StartArguments>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+  </PropertyGroup>
   <ItemGroup>
     <Reference Include="Open.Nat">
       <HintPath>..\thirdparty\download\Open.Nat.dll</HintPath>

warning: CRLF will be replaced by LF in OpenRA.Game/OpenRA.Game.csproj.
The file will have its original line endings in your working directory

```

Reply:
Context from IRC: the optimize flag causes issues with debugging on windows, whereas Mono seems to be smart enough to avoid them (or more likely it automatically disables optimization at runtime) when running in a debugger.

I'll change the csproj  files to disable optimizations in the Debug configuration, and have the makefile builds force Release.